Library/Formula/bazel.rb

```diff
@@ -0,0 +1,23 @@
+class Bazel < Formula
+  desc "Google's own build tool"
+  homepage "http://bazel.io/"
+  url "https://github.com/bazelbuild/bazel/releases/download/0.1.1/bazel-0.1.1-installer-darwin-x86_64.sh"
```

Comment:
@mikemcquaid building from source seems to be a bit confusing on where the built files etc. are... I could look into it, but the binary version I think is now working with better tests... could you kindly have a look?